views/templates/admin/new_release.tpl

```diff
@@ -35,7 +35,7 @@
         <div class="alert alert-warning">
             <button type="button" class="close js-mollie-upgrade-tip-close" data-dismiss="alert">×</button>
             <ul class="list-unstyled">
-                {$github_url|escape:'html':'UTF-8'}
+                {$github_url}
```

Comment:
does this github_url prints html ? 

like

```html
<a href="http://myrandomurl.com"></a>
```

?

Reply:
`$this->l('You are currently using version \'%s\' of this plugin. The latest version is \'%s\'. We advice you to [1]update[/1] to enjoy the latest features. '),` and then into [1]update[1] inserts github_url. 
All this is required to have single translation instead of 3